docs/guides/quickstart-dfp.md

```diff
@@ -0,0 +1,258 @@
+---
+page_title: "Managing Policy Based DFP service using the BloxOne Terraform Provider"
+subcategory: "Guides"
+description: |-
+  This guide provides step-by-step instructions for using the BloxOne Terraform Provider to manage the DFP Service , Security Policies and various resources associated to it.
+---
+
+# Managing Policy Based DFP service using the BloxOne Terraform Provider
+
+This guide provides step-by-step instructions for using the BloxOne Terraform Provider to manage the DFP Service , Security Policies and various resources associated to it.
+
+## Configuring the Provider
+
+The provider needs to be configured with an API key and the URL of the Infoblox Cloud Services Portal (CSP). You can get the API Key from the Infoblox Cloud Services Portal (CSP) by following the steps outlined in this guide - [Configuring User API Keys](https://docs.infoblox.com/space/BloxOneCloud/35430405/Configuring+User+API+Keys).
+
+Create a directory for the Terraform configuration and create a file named `main.tf` with the following content:
+
+````terraform
+terraform {
+  required_providers {
+    bloxone = {
+      source  = "infobloxopen/bloxone"
+      version = ">= 1.0.0"
+    }
+  }
+  required_version = ">= 1.5.0"
+}
+
+provider "bloxone" {
+  csp_url = "https://csp.infoblox.com"
```

Comment:
We suggest to use environment variables, if then I would recommend to update the Terraform content with ENV variables for both CSP & API.   Then add some comment for ENV variable with value.

Reply:
Users generally provide credentials as Env variables . This is given just as an example here . We have also added a warning against hard coded credentials below !